src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <ul class="calendar calendar--start-day-sun calendar--month-length-31">
+      <li class="calendar__day calendar__day--1"></li>
+      <li class="calendar__day calendar__day--2"></li>
+      <li class="calendar__day calendar__day--3"></li>
+      <li class="calendar__day calendar__day--4"></li>
+      <li class="calendar__day calendar__day--5"></li>
+      <li class="calendar__day calendar__day--6"></li>
+      <li class="calendar__day calendar__day--7"></li>
+      <li class="calendar__day calendar__day--8"></li>
+      <li class="calendar__day calendar__day--9"></li>
+      <li class="calendar__day calendar__day--10"></li>
+      <li class="calendar__day calendar__day--11"></li>
+      <li class="calendar__day calendar__day--12"></li>
+      <li class="calendar__day calendar__day--13"></li>
+      <li class="calendar__day calendar__day--14"></li>
+      <li class="calendar__day calendar__day--15"></li>
+      <li class="calendar__day calendar__day--16"></li>
+      <li class="calendar__day calendar__day--17"></li>
+      <li class="calendar__day calendar__day--18"></li>
+      <li class="calendar__day calendar__day--19"></li>
+      <li class="calendar__day calendar__day--20"></li>
+      <li class="calendar__day calendar__day--21"></li>
+      <li class="calendar__day calendar__day--22"></li>
+      <li class="calendar__day calendar__day--23"></li>
+      <li class="calendar__day calendar__day--24"></li>
+      <li class="calendar__day calendar__day--25"></li>
+      <li class="calendar__day calendar__day--26"></li>
+      <li class="calendar__day calendar__day--27"></li>
+      <li class="calendar__day calendar__day--28"></li>
+      <li class="calendar__day calendar__day--29"></li>
+      <li class="calendar__day calendar__day--30"></li>
+      <li class="calendar__day calendar__day--31"></li>
```

Comment:
It seems like you are manually adding each day of the month. Consider using a template engine or server-side code to generate this list dynamically to reduce the amount of HTML and make it easier to maintain.

Reply:
we don't use any html templates in this task